plugins/codemirror/dialogs/codemirrordialog.js

```diff
@@ -43,7 +43,15 @@ CKEDITOR.dialog.add('codemirrordialog', function (editor) {
 			preview.setSize('width', defaultWidth);
 
 			var iframe = this._createContentIframe(preview);
-			var iframeBody = iframe.$.contentDocument.body;
+
+			var iframeWindow = iframe.$.contentWindow;
+
+			var iframeBody = iframeWindow.document.body;
+
+			var iframeDocument = iframeWindow.document;
+
+			iframeDocument.open();
+			iframeDocument.close();
```

Comment:
@mateomustapic what @carloslancha mentioned [here](https://github.com/liferay/liferay-ckeditor/pull/131#issuecomment-692711583) is also what I had in mind and for me it makes sense.
I tried with 

```js
iframe.$.contentWindow.addEventListener('load', function() {
  var iframeBody = iframe.$.contentDocument.body;
  iframeBody.innerHTML = codeMirrorEditor.getValue();
});
``` 
and that works for me

Reply:
hey @julien, Thanks for this. 
Yesterday at that moment, I didn't have better solution than the one I provided.
Having in mind what Carlos mentioned earlier, I will modify and push the change with this code snippet you provided.

Reply:
@mateomustapic no problem, that's why we do reviews, just make sure that what I suggested works